index.js

```diff
@@ -31,6 +31,12 @@ var isPlainObject = function isPlainObject(obj) {
 	return typeof key === 'undefined' || hasOwn.call(obj, key);
 };
 
+var isBuffer = function isBuffer(buff) {
```

Comment:
this implementation won't work across realms - ie, iframes or the "vm" module. Any reliable check can't use `instanceof`. `Buffer.isBuffer` may work but it'd be better to implement a more reliable check.